message-index/messages/GHC-84077/example-parsing/before/index.md

```diff
@@ -0,0 +1,16 @@
+---
+ title: Missing space in type application
+ ---
+
+ ## Error Message
+
+ ```
+<interactive>:84:6: error:
```

Comment:
Shouldn't the message here be the one from using GHC on the example file? Then it will include the code and the line numbers and such.

Reply:
applied